src/style-spec/expression/definitions/in.js

```diff
@@ -42,7 +42,7 @@ class In implements Expression {
         const needle = (this.needle.evaluate(ctx): any);
         const haystack = (this.haystack.evaluate(ctx): any);
 
-        if (!haystack) return false;
+        if (haystack == null) return false;
```

Comment:
Should this be triple equals? Will == on an empty string evaluate as false?

Reply:
This is a way of detecting `undefined` and `null` values while allowing other falsey values to pass through.

```
undefined == null // true
null == null // true
false == null // false
'' == null // false
```